docgenhtml/plugins/org.polarsys.capella.docgen/src/org/polarsys/capella/docgen/util/ImageHandlerService.java

```diff
@@ -0,0 +1,49 @@
+/*******************************************************************************
+ * Copyright (c) 2024 THALES GLOBAL SERVICES.
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *  
+ * Contributors:
+ *   Thales - initial API and implementation
+ ******************************************************************************/
+package org.polarsys.capella.docgen.util;
+
+import java.util.LinkedHashSet;
+import java.util.List;
+
+import org.eclipse.core.resources.IFolder;
+import org.eclipse.core.runtime.ILog;
+import org.eclipse.emf.ecore.EObject;
+
+public class ImageHandlerService {
+
+  public static ImageHandlerService eINSTANCE = new ImageHandlerService();
+
+  private LinkedHashSet<IImageHandler> imageHandlers = new LinkedHashSet<>();
+  private IImageHandler defaultHandler = new DefaultImageHandler();
+
+  private ImageHandlerService() {
+    initializeServices();
+  }
+
+  private void initializeServices() {
+    imageHandlers.add(new CDOImageHandler());
```

Comment:
since there is only one handler, we can remove the hashSet and use CDOImageHandler directly